<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MUL8_MainView</class>
 <widget class="QMainWindow" name="MUL8_MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>701</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Main">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QPushButton" name="playMotionButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>380</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Play Motion</string>
      </property>
     </widget>
     <widget class="QListWidget" name="preExistingMotionList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>261</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="currentMotionPlaybackList">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>170</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>180</y>
        <width>20</width>
        <height>161</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="addMotionToListButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Motion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeMotionFromListButton">
      <property name="geometry">
       <rect>
        <x>493</x>
        <y>120</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Motion</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>360</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Accept</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearListButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>380</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Edit">
     <attribute name="title">
      <string>Edit</string>
     </attribute>
     <widget class="QGroupBox" name="MotorControlBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>591</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>Motor Positions</string>
      </property>
      <widget class="QLabel" name="m24Label">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>62</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>24:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m23Label">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>40</y>
         <width>62</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>23:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m15Label">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>60</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>15:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m16Label">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>60</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>16:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m13Label">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>60</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>13:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m14Label">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>60</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>14:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m17Label">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>120</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>17:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m18Label">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>120</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>18:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m19Label">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>180</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>19:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m20Label">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>180</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>20:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m21Label">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>200</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>21:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m22Label">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>200</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>22:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m1Label">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>190</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>1:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m2Label">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>190</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>2:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m3Label">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>210</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>3:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m4Label">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>210</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>4:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m5Label">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>250</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>5:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m7Label">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>270</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>7:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m6Label">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>250</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>6:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m8Label">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>270</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>8:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m9Label">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>330</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>9:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m11Label">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>310</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>11:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m12Label">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>310</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>12:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m10Label">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>330</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>10:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QGroupBox" name="motorCommandBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>181</width>
         <height>371</height>
        </rect>
       </property>
       <property name="title">
        <string>Motor Commands</string>
       </property>
       <widget class="QPushButton" name="nextStepButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>114</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Next Step</string>
        </property>
       </widget>
       <widget class="QPushButton" name="releaseMotorButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>260</y>
          <width>114</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Release Motor</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="releaseMotorSpinBox">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>260</y>
          <width>49</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
       </widget>
       <widget class="QPushButton" name="releaseAllMotorButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>141</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Release All Motors</string>
        </property>
       </widget>
       <widget class="QPushButton" name="engageMotorButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>190</y>
          <width>114</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Engage Motor</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="engageMotorSpinBox">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>190</y>
          <width>49</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
       </widget>
       <widget class="QPushButton" name="engageAllMotorButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>141</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Engage All Motors</string>
        </property>
       </widget>
       <widget class="QPushButton" name="releaseLeftLegButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>141</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Release Left Leg</string>
        </property>
       </widget>
       <widget class="QPushButton" name="releaseRightLegButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>141</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Release Right Leg</string>
        </property>
       </widget>
       <widget class="QPushButton" name="engageLeftLegButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>141</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Engage Left Leg</string>
        </property>
       </widget>
       <widget class="QPushButton" name="engageRightLegButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>141</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Engage Right Leg</string>
        </property>
       </widget>
       <widget class="QPushButton" name="savePositionsButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>330</y>
          <width>141</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Save Positions</string>
        </property>
       </widget>
       <widget class="QLabel" name="stepNumLabel">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>20</y>
          <width>21</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="motorValueBox">
       <property name="geometry">
        <rect>
         <x>209</x>
         <y>270</y>
         <width>131</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>Motor Values</string>
       </property>
       <widget class="QRadioButton" name="motorTempRadioButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Motor Temp</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="motorPosRadioButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Motor Position</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="robotLabel">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>120</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Robot Motors</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMU_L8_Motion_Editor">
    <property name="title">
     <string>MU-L8 Motion Editor</string>
    </property>
   </widget>
   <addaction name="menuMU_L8_Motion_Editor"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
